applications/app/views/fragments/newGalleryBody.scala.html

```diff
@@ -0,0 +1,127 @@
+
+    <article id="article" class="@RenderClasses(
+            Map("content--advertisement-feature" -> gallery.commercial.isAdvertisementFeature,
+                "paid-content--advertisement-feature" -> gallery.commercial.isAdvertisementFeature
+            ),
+            "content", "content--media", "content--gallery", "tonal", "new-gallery", "content--immersive", s"tonal--${toneClass(gallery)}"
+        )"
+        itemscope itemtype="@gallery.metadata.schemaType" role="main">
+
+        @if(gallery.commercial.isAdvertisementFeature){
+            @fragments.guBand()
+        }
+
+        @fragments.galleryHeader(gallery, gallery.commercial.isSponsored(Some(Edition(request))) || gallery.commercial.isFoundationSupported || gallery.commercial.isAdvertisementFeature)
+
+        <div class="content__main tonal__main tonal__main--@toneClass(gallery)">
+            <div class="gs-container">
+                <div class="content__main-column content__main-column--gallery">
+                    @fragments.witnessCallToAction(gallery.content)
+                    <ul class="gallery">
+                        @gallery.lightbox.largestCrops.zipWithRowInfo.map { case (image, row) =>
+                            @* Skipping first image, as that is used for the background*@
+                            @if(row.rowNum != 1) {
+                                @galleryItem(Seq("inline1", "inline2"), 4, image, row.rowNum-1, gallery.lightbox.imageContainer(row.rowNum - 1))
+                            }
+                        }
+                    </ul>
+                    @fragments.witnessCallToAction(gallery.content)
+                    @fragments.submeta(gallery)
+                </div>
+            </div>
+        </div>
+    </article>
+
+    @if(gallery.content.showInRelated &&
+        !gallery.commercial.isAdvertisementFeature ) {
+        <div class="gallery__most-popular facia-container fc-container--media hide-on-childrens-books-site">
+            <div class="js-gallery-most-popular tonal--@toneClass(gallery)">
+                <div class="fc-container">
+                    <div class="fc-container__inner">
+                        <div class="fc-container__header">
+                            <h2 class="fc-container__header__title">
+                                <a class="most-viewed-no-js tone-colour" href="@LinkTo{/gallery/most-viewed}" data-link-name="Most viewed galleries">More galleries</a>
+                            </h2>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    }
+    @fragments.contentFooter(gallery, related, "media")
+</div>
+
+@galleryItem(adSlots: Seq[String], adInterval: Int, image: model.ImageAsset, rowNum: Int, imageElement: model.ImageElement) = {
+
+    <li id="img-@rowNum" class="gallery__item js-gallery-item" data-link-name="Gallery item | @rowNum">
+        <figure itemscope itemtype="http://schema.org/ImageObject">
```

Comment:
i think this will be the third instance of very similar code in the codebase, is it similar enough to reuse an existing image fragment? or if not, should they be made similar enough and reused?